deepspeed/model_implementations/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .diffusers.unet import DSUNet
```

Comment:
We can't import this unless `diffusers` is available, since it's not a hard dependency of deepspeed. Similar for `transformers` we cannot assume it's installed

Reply:
Thanks! We don't need to import it in the __init__ so I removed it. The other two are not importing any external dependencies so they are okay to keep.